dsl_parser/elements/imports.py

```diff
@@ -253,7 +258,10 @@ def _merge_into_dict_or_throw_on_duplicate(from_dict_holder, to_dict_holder,
                    "on '{1}'".format(key_name, key_holder.value))
 
 
-def _fetch_import(import_url):
+def _fetch_import(import_url, resolver):
+    url_parts = import_url.split(':')
+    if url_parts[0] in ['http', 'https', 'ftp']:
+        return resolver.resolve(import_url)
```

Comment:
I want to suggest to all import resolution will go through the resolved without checking the for http/https/ftp protocol. This will also arbitrarily defined imports to be supported by custom resolvers. So maybe the below fallback code should not be here at all?